src/containers/pages/PlanAssignment/helpers/JurisdictionAssignmentForm/helpers.ts

```diff
@@ -55,7 +55,7 @@ export const getPayload = (
   }
 
   // turns out if you put it in the loop it keeps subtracting a day for every iteration
-  const retireDate = now.format();
+  const retireDate = moment(new Date()).format();
```

Comment:
@ciremusyoka isn't `now === moment(new Date())`. ama whats the change?

Reply:
@p-netm [this](https://github.com/onaio/reveal-frontend/blob/4c4413d9a3a92b3dfc2f1e21274ed2f708dd3b59/src/containers/pages/PlanAssignment/helpers/JurisdictionAssignmentForm/helpers.ts#L31-L33) part is updating `now` date object that why am redefining it.

Reply:
cool